src/core/Wizard/Wizard.tsx

```diff
@@ -55,43 +55,43 @@ const defaultWizardLocalization: WizardLocalization = {
 };
 
 /**
- * A Wizard.
- * Give it some steps.
- * Tell it what step to display.
- * Set the type if you'd like.
- * That's about it.
+ * A wizard displays progress through a sequence of logical and numbered steps.
```

Comment:
Kinda liked that previous description.. Was fun to read..But maybe this is more professional.